_contributors/jburge.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Justin Burge
+---
+## Affiliation
+Hutch Data Commonwealth
+
+
+## Title
+Training/Education Specialist
+
+
+## Contribution Domain and Type
+Data Science
```

Comment:
You mean Scientific Computing?  And Writing/Editing?

Reply:
works for me. I got that originally from outlook.